app/assets/javascripts/slotcars/shared/lib/drivable.js.coffee

```
@@ -0,0 +1,20 @@
+Shared.Drivable = Ember.Mixin.create
+
+  speed: 0
+  maxSpeed: 0
+
+  acceleration: 0
+  deceleration: 0
+
```

For reasons of consistency we should throw an exception if the `Movable` mixin is not provided ...